src/app.js

```diff
@@ -27,28 +27,79 @@ export default class App extends React.Component {
 
   }
 
+  homepage(event) {
+    this.setState({page: 'users'}) // Trying to link back to page of users
+  }
+
   render() {
     if(this.state.page == 'login') {
       return(
           <div className = 'container'>
-            <h1>Connect As: </h1>
-            <div className="btn-group" role = "group">
-            <button type="button" className = "btn btn-secondary" id = "tbutton" onClick = { this.connect }>Tourist</button>
-            <button type="button" className = "btn btn-secondary" id = "gbutton" onClick = { this.connect }>Guide</button>
+            <div className="navbar navbar-default">
+              <div className="container-fluid">
+                <div className="navbar-header">
+                  <p id="logo">Wander</p>
+                </div>
+              </div>
+            </div>
+            <h2 id="title">Welcome to Wander! </h2>
+            <div className = 'col-sm-12 col-md-6'>
+              <img src="img/image.png" id="home_img"></img>
+            </div>
+            <div className = 'col-sm-12 col-md-6'>
+              <form>
+                <div className="form-group row">
+                    <i className="fa fa-user" aria-hidden="true"></i>
+                    <input type="text" id = "username_input" className="form-control" placeholder="Username"></input>
+                    <i className="fa fa-key" aria-hidden="true"></i>
+                    <input type="text" id = "pw_input" className="form-control" placeholder="Password"></input>
+                </div>
+              </form>
+                <div className = 'col-sm-12 col-md-6'>
+                  <button type="button btn-block" className = "btn btn-secondary" id = "tbutton" onClick = { this.connect }>Log-in as Tourist</button>
+                </div>
+                <div className = 'col-sm-12 col-md-6'>
+                  <button type="button btn-block" className = "btn btn-secondary" id = "gbutton" onClick = { this.connect }>Log-in as Guide</button>
+                </div>
             </div>
           </div>
         )
     } else if (this.state.page == 'loading') {
       return(
-        <center>
-          <i className="fa fa-repeat" aria-hidden="true"></i>
-        </center>
+        <div className = 'container'>
+          <div className="navbar navbar-default">
+            <div className="container-fluid">
+              <div className="navbar-header">
+                <p id="logo">Wander</p>
+              </div>
+            </div>
+          </div>
+          <center>
+            <i className="fa fa-repeat" aria-hidden="true"></i>
+          </center>
+        </div>
       )
     } else {
       return (
-        <Users role = { this.state.role }
-               view = { 'users' }
-               position = { this.state.position }/>
+        <div className = 'container'>
+          <div className="navbar navbar-default">
```

Comment:
You repeat the `navbar` piece twice, just break it out to a `Navigation` component and then place it in here *once*